src/App.js

```diff
@@ -97,6 +99,8 @@ const App = () => {
           {/* <Route path="/addbook" element={<AddBook/>}/> */}
           <Route path="/SearchResultsBook" element={<SearchResultsBook />} />
           <Route path="/ReaderQandA/:aid" element={<ReaderQandA />} />
+          
+          <Route path="/AboutPage" element={<AboutPage />} />
```

Comment:
Can we make this route just /about or something similar?

Reply:
resolved this, i renamed it to /About